vita3k/modules/SceLibKernel/SceLibKernel.cpp

```diff
@@ -1108,8 +1134,15 @@ EXPORT(int, sceKernelGetThreadEventInfo) {
     return UNIMPLEMENTED();
 }
 
-EXPORT(int, sceKernelGetThreadExitStatus) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceKernelGetThreadExitStatus, SceUID thid) {
```

Comment:
We're erasing thread from host.kernel.threads in some functions. We need to make sure we don't delete them.

Reply:
This is maybe managed by user application. I'm seeing sceKernelExitDeleteThread and sceKernelDeleteThread. After all, it will cause error if it didn't find it. We can see how things turn out after merging this.

Reply:
As told here https://wiki.henkaku.xyz/vita/SceKernelThreadMgr#sceKernelExitThread 
 * The thread exit status is saved while the thread exits and is in the DORMANT state,
 * but the value is lost if the thread is restarted or deleted with sceKernelStartThread().

I've seen using this function to check thread status (running or finished and deleted)

Reply:
I see so it's supposed to be managed by application.